app/controllers/hq/partners_controller.rb

```diff
@@ -41,6 +51,9 @@ def save_partner
     if @partner.save
       flash[:success] = 'Partner successfuly saved'
       redirect_to hq_partner_url(@partner)
+    else
+      load_associations
+      false
```

Comment:
This awkward explicit return of `false` made me wonder whether instead it may be cleaner to use `save_parter or re_render 'view'`, with
```ruby
def re_render(view)
  load_associations
  render view
end
```